libs/ActivityKit/source/activities/SuperSimon.cpp

```diff
@@ -0,0 +1,112 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+// LCOV_EXCL_START
+
+#include "SuperSimon.h"
+
+#include "LogKit.h"
+
+namespace leka::activity {
+using namespace std::chrono;
+
+void SuperSimon::start()
+{
+	_score		  = 0;
+	_pos		  = 0;
+	_index_number = 0;
+
+	_videokit.displayImage("fs/home/img/system/robot-face-smiling-slightly.jpg");
+
+	_backup_callback = _rfidkit.getCallback();
+	std::shuffle(_colors_span.begin(), _colors_span.end(), std::mt19937(time(nullptr)));
+	rtos::ThisThread::sleep_for(1s);
+
+	setNextDisplay();
+
+	auto on_tag_detected_callback = [this](const MagicCard &card) {
+		auto isCardANumber = [](const MagicCard &card) {
+			return card == MagicCard::number_1 || card == MagicCard::number_2 || card == MagicCard::number_3 ||
+				   card == MagicCard::number_4 || card == MagicCard::number_5 || card == MagicCard::number_6 ||
+				   card == MagicCard::number_7 || card == MagicCard::number_8 || card == MagicCard::number_9 ||
+				   card == MagicCard::number_10;
```

Comment:
You have the `numbers_table`, why not use `std::find`?

https://stackoverflow.com/a/64341713/2205264